keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -97,6 +97,16 @@ The second iteration, `kubescheduler.config.k8s.io/v1beta2`, includes the follow
     - `RequestedToCapacityRatio` (in favor of `NodeResourcesFit` plugin with a `RequestedToCapacityRatio` scoring strategy)
   - Cleanup of validation hacks.
 
+The third iteration, `kubescheduler.config.k8s.io/v1beta3`, includes the following changes:
+  - Change the weight of plugins that can be influenced by end users through the Pod specs.
+    - `InterPodAffinity` to 2
+    - `NodeAffinity` to 2
+    - `TaintToleration` to 3 as the usage of node tainting to group nodes in the cluster is a increasingly becoming a use-case 
+       for many user workloads
```

Comment:
why 2 for pod/node affinity and 3 for the taint? did we run some experiments to find a good weight?

Reply:
Two reasons:

- We used to have `1000` nodePreferAvoidPods in earlier versions of API and we've deprecated it in favor of using taints/tolerations
- Taints/tolerations is becoming a strong use-case for grouping nodes in the clusters as mentioned. However, the value of 3 is not fixed, we'll write tests to ensure that the values are in fact helping in scheduling pods onto appropriate nodes and make changes accordingly.